src/pages/ContributionsPage/ContributionsPage.tsx

```diff
@@ -351,32 +403,57 @@ const ContributionsPage = () => {
           }}
         >
           <Text h1>Contributions</Text>
-          <Button
-            outlined
-            color="primary"
-            onClick={() => {
-              setCurrentContribution({
-                contribution: getNewContribution(
-                  currentUserId,
-                  memoizedEpochData.contributions[0]
-                ),
-                epoch: getCurrentEpoch(memoizedEpochData.epochs),
-              });
-              resetField('description', { defaultValue: '' });
-              resetCreateMutation();
-              setModalOpen(true);
-            }}
-          >
-            Add Contribution
-          </Button>
+          {!editHelpText ? (
+            isAdmin && (
+              <Button
+                outlined
+                color="primary"
+                type="submit"
+                onClick={() => {
+                  setEditHelpText(true);
+                }}
+              >
+                Edit Help Text
+              </Button>
+            )
+          ) : (
+            <Button
+              outlined
+              color="primary"
+              type="submit"
+              onClick={handleSubmit(onSubmit)}
+            >
+              Save
+            </Button>
+          )}
         </Flex>
-        <Text p>What have you been working on?</Text>
+        <Box>
+          {!editHelpText ? (
+            <Text p>
+              {updatedTeamSelText || 'What have you been working on?'}
```

Comment:
i see that you've initialized `updatedTeamSelText` with `data?.circles_by_pk?.team_sel_text`. However, this doesn't work as desired. I can see in my [testing](https://coordinape-d47jepbje-coordinape.vercel.app/) that it's showing "What have you been working on?" even though there is a `team_sel_text` value.

This is why:

1. Component starts rendering
2. useQuery starts fetching data
3. useState initializes `updatedTeamSelText` to a blank value
4. useQuery returns data and component re-renders. `data?.circles_by_pk?.team_sel_text` now has a value. but `updatedTeamSelText` does _not_ change, because `setUpdatedTeamSelText` was not called.

```suggestion
              {updatedTeamSelText || data?.circles_by_pk?.team_sel_text || 'What have you been working on?'}
```

Reply:
I changed the logic here a bit